### PR TITLE
taxii c2c doc added

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md [hunk 1]
```diff
@@ -402,6 +402,12 @@ In this section, we'll introduce the following concepts:
         <p>Learn to receive WSS Access logs from the Symantec WSS API.</p>
         </div>
       </div>
+      <div className="box smallbox card">
+        <div className="container">
+        <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/taxii"><img src={useBaseUrl('img/send-data/taxii_logo.png')} alt="icon" width="60"/><h4>TAXII</h4></a>
+        <p>Learn to collect indicators from threat intel services using the TAXII/STIX 1.1 specification.</p>
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md:408]: If you run a search for this topic, looks like the preferred nomenclature is STIX/TAXII 

```suggestion
        <p>Learn to collect indicators from threat intel services using the STIX/TAXII 1.1 specification.</p>
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/taxii.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+---
+id: taxii
+title: TAXII Source
+sidebar_label: TAXII
+tags:
+  - cloud-to-cloud
+  - taxii
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/taxii.md:7]:
```suggestion
  - taxii
  - stix
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/taxii.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+import CodeBlock from '@theme/CodeBlock';
+import ExampleJSON from '/img/c2c/taxii/example.json';
+import MyComponentSource from '!!raw-loader!/img/c2c/taxii/example.json';
+import TerraformExample from '!!raw-loader!/img/c2c/taxii/example.tf';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/taxii_logo.png')} alt="icon" width="50"/>
+
+The TAXII source provides a secure endpoint to collect indicators from threat intel services using the TAXII/STIX 1.1 specification and send them to Sumo Logic Threat Intelligence.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/taxii.md:19]:
```suggestion
The Sumo Logic TAXII source provides a secure endpoint to collect indicators from threat intel services using the STIX/TAXII 1.1 specification and sends them to Sumo Logic Threat Intelligence.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/taxii.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+| Polling Interval | Data |
+| :--- | :--- |
+| 1 hour |  Indicators |
+
+## Setup
+
+### Vendor configuration
+
+
+
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/taxii.md:36]: should this be blank?

File: static/img/c2c/taxii/example.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (kimsauce) [inline static/img/c2c/taxii/example.json:1]: As this is a JSON file, it should be moved to the `static/files` folder (currently it's in the images folder).

File: static/img/c2c/taxii/example.tf [hunk 6]
```diff
@@ -0,0 +1,19 @@
+resource "sumologic_cloud_to_cloud_source" "taxii_source" {
```
Comment (kimsauce) [inline static/img/c2c/taxii/example.tf:1]: As this is a .tf file, it should be moved to the `static/files` folder (currently it's in the images folder).

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/taxii.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
+---
+id: taxii
+title: TAXII Source
+sidebar_label: TAXII
+tags:
+  - cloud-to-cloud
+  - taxii
+description: The TAXII source provides a secure endpoint to collect indicators from threat intel services using the TAXII/STIX 1.1 specification.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/taxii.md:8]:
```suggestion
description: The TAXII source provides a secure endpoint to collect indicators from threat intel services using the STIX/TAXII 1.1 specification.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
a few changes

File: static/img/c2c/taxii/example.json (hunk 5 above)
Comment (JV0812) [reply static/img/c2c/taxii/example.json:1]: You are correct Kim, it should be under static/files. But when we did the C2C cleanup, we moved all these examples to images folder by mistake. 
I will take this as a separate task and will move all the existing .json and .tf examples from img folder to  files folder.

File: static/img/c2c/taxii/example.tf (hunk 6 above)
Comment (JV0812) [reply static/img/c2c/taxii/example.tf:1]: You are correct Kim, it should be under static/files. But when we did the C2C cleanup, we moved all these examples to images folder by mistake. 
I will take this as a separate task and will move all the existing .json and .tf examples from img folder to  files folder.

Comment (kimsauce) [reply static/img/c2c/taxii/example.tf:1]: That sounds good, thanks for handling! Here's the ticket i've made. and i'll approve this PR https://app.asana.com/0/1183741177809740/1206145385179444/f

File: static/img/c2c/taxii/example.json (hunk 5 above)
Comment (kimsauce) [reply static/img/c2c/taxii/example.json:1]: https://github.com/SumoLogic/sumologic-documentation/pull/3388#discussion_r1423053696

**Review by kimsauce — CHANGES_REQUESTED** [review]
JV is waiting on https://github.com/SumoLogic/sumologic-documentation/pull/3388#discussion_r1422652085

**Comment by JV0812** [comment]
As per the comment in [this](https://sumologic.atlassian.net/browse/CONN-2743?focusedCommentId=504246) JIRA ticket, there is no need to create a public doc for TAXII C2C now. So closing this PR.

**Pull request closed without merging**